network/transport/v2/protocol.go

```diff
@@ -237,7 +237,16 @@ func (p *protocol) gossipTransaction(event dag.Event) (bool, error) {
 }
 
 func (p *protocol) sendGossip(id transport.PeerID, refs []hash.SHA256Hash, xor hash.SHA256Hash, clock uint32) bool {
-	if err := p.sendGossipMsg(id, refs, xor, clock); err != nil {
+	conn := p.connectionList.Get(grpc.ByConnected(), grpc.ByPeerID(id))
```

Comment:
missing filter on nodeDID?

Reply:
gossip manager and conversation manager still track connections based on peerID, created #1950 for this